torch/_torch_docs.py

```diff
@@ -7385,6 +7385,45 @@ def merge_dicts(*dicts):
             [3, 2, 1, 0]])
 """.format(**common_args))
 
+add_docstr(torch.msort,
+           r"""
+msort(input, descending=False, *, out=None) -> (Tensor, LongTensor)
+
+Sorts the elements of the :attr:`input` tensor along the first dimension
+in ascending order by value.
+
+If :attr:`descending` is ``True`` then the elements are sorted in descending
+order by value.
+
+A namedtuple of (values, indices) is returned, where the `values` are the
+sorted values and `indices` are the indices of the elements in the original
+`input` tensor.
+
+.. note:: `torch.msort(a)` is equivalent to `torch.sort(a, dim=0)`.
```

Comment:
This will need a small update to be equivalent to `torch.sort(t, dim=0)[0]`

Reply:
Addressed!